### PR TITLE
Add small summary to the Readme and fix typo

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -6,6 +6,11 @@ server is not available. DDHCP allows every realm to have its own server,
 while preserving roaming features for the client, by maintaining a common
 consens about lease state.
 
+The IPv4 address range is divided into many small blocks. If a router wants to
+assign an address to a client, it grabs a small block from the pool and 
+distributes an address to the client. The routers agree among themselves which
+blocks are used or free.
+
```
Comment (sargon) [inline README]: That's not entirely correct, it currently only serves from already reserved blocks, the mechanism which reserve new blocks only looks at the currently available free leases and how many free leases should be available,

**Comment by rubo77** [comment]
done

**Comment by sargon** [comment]
I decided against merging, since I think those lines belong into an algorithm explanation and
not into the deamon readme.

**Comment by rubo77** [comment]
And where should we add the algorithm explanation?

**Pull request closed without merging**